src/app/clusters/operational-state-server/operational-state-server.h

```diff
@@ -140,111 +204,66 @@ class OperationalStateServer : public CommandHandlerInterface, public AttributeA
  */
 class Delegate
 {
+protected:
+    Instance * mServer = nullptr;
```

Comment:
Do you mean `const Instance* GetServer() { return mServer; }`? This is not possible because the subclasses will need to call `Set...` methods which modify private members.

Reply:
No, I mean `Instance * GetServer() const { return mServer; }`